share/harvest/scheduler.py

```diff
@@ -34,12 +34,12 @@ def all(self, cutoff=None, allow_full_harvest=True, **kwargs):
         if hasattr(self.source_config, 'latest'):
             latest_date = self.source_config.latest
         else:
-            latest_date = HarvestLog.objects.filter(source_config=self.source_config).aggregate(models.Max('end_date'))
+            latest_date = HarvestLog.objects.filter(source_config=self.source_config).aggregate(models.Max('end_date'))['end_date__max']
```

Comment:
Since you did it below, what about `self.source_config.harvest_logs.aggregate...`?